app.py

```diff
@@ -0,0 +1,138 @@
+import base64
+import json
+import logging
+import os
+import sys
+import boto3
+
+from chalice import Chalice
+
+app_name = 'FilterArcEventsApp'
+
+app = Chalice(app_name=app_name)
+
+# Initialize the logger
+logging.basicConfig(filename='app.log', filemode='w', format='%(name)s - %(levelname)s - %(message)s')
```

Comment:
We can remove the custom log file as it is not needed

Reply:
done